fedimint-cli/src/client.rs

```diff
@@ -101,6 +101,9 @@ pub enum ClientCmd {
         /// Amount to pay, used for lnurl
         #[clap(long)]
         amount: Option<Amount>,
+        /// Invoice comment/description, used on lnurl
+        #[clap(long)]
+        comment: Option<String>,
```

Comment:
would it make sense to call this `lnurl_comment`? just to make it clearer that this is an lnurl-specific thing?

Reply:
Not sure it would cause confusion, but yeah, we can change that.